recipes/compyle/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "compyle" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: "./pypr-compyle-v0.5-112-g42a7868.tar.gz"
+  sha256: "eceb3bc0970b570056138174e77a2eb0d6be2128c81a7315436bb996c668bbd3"
+
+build:
+  number: 0
+  script: " {{ PYTHON }} -m pip install ./pypr-compyle-v0.5-112-g42a7868.tar.gz --no-deps --ignore-installed -vv "
```

I guess after moving to the github tarball this will be:
```suggestion
  script: " {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

---

The issue is that you downloading the HTML of the page instead of a tarball. If a tarball is downloaded conda will automatically unpack it.